backend/src/app/rpc/commands/files_thumbnails.clj

```diff
@@ -205,14 +206,18 @@
 
                        file (bfc/get-file cfg file-id
                                           :realize? true
-                                          :read-only? true)]
+                                          :read-only? true)
+
+                       strip-frames-with-thumbnails
+                       (or (nil? strip-frames-with-thumbnails) ;; if not present, default to true
+                           (= strip-frames-with-thumbnails "true"))]
```

Comment:
The value of `strip-frames-with-thumbnails` will be decoded to boolean automatically because of schema so, this probably should be `(true? strip-frames-with-thumbnails)`